packages/client/components/ReflectionCard/AddReactjiButton.tsx

```diff
@@ -38,7 +38,7 @@ const AddReactjiButton = (props: Props) => {
     onToggle(emojiId)
   }
   return (
-    <Popover.Root open={open} onOpenChange={onOpenChange}>
+    <Popover.Root open={open} onOpenChange={onOpenChange} modal>
```

Comment:
There seems to be a bug in the controlling behaviour, removing `onOpenChange` instead would have the same effect and still properly open and close.